modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -16,6 +16,15 @@
 		"issueKey": "LPD-7751",
 		"to": "fetchCommerceCatalogByExternalReferenceCode(param#1#, param#0#)"
 	},
+	{
+		"from": "serviceLocator.findService(\"com.liferay.dynamic.data.mapping.storage.StorageEngine\")",
+		"issueKey": "LPD-7762",
+		"skipParametersValidation": true,
+		"to": "staticUtil[\"com.liferay.dynamic.data.mapping.kernel.StorageEngineManagerUtil\"]",
+		"validExtensions": [
+			"ftl"
+		]
+	},
```

Comment:
Hey Drew I did searching for this and we dont have any usages. So itll be whatever the client code declares the variable name as which I dont think has a high chance of being storageEngine, itll be whatever their SF standard is. I think this might inconvenience clients more than aid them if that makes sense. Do you think I should still change the variable name? I think it was just an example shown

Pasted from Slack for visibility @drewbrokke

Reply:
No we can leave it, thank you for the explanation, @kyle-miho